src/Utilities/ProcessExtensions.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Utilities
+{
+    internal static class ProcessExtensions
+    {
+#if !RUNTIME_TYPE_NETCORE
```

Comment:
For a Mono build, this would be true and so `_isWindows=true`, which would be incorrect. Mono build of msbuild could run on !windows platforms too!